backend/.env.example

```
@@ -96,14 +98,21 @@ WEBHOOK_HEADERS=
+# AES settings are read as UTF-8 strings by the app.
+# Keys must be 32 characters (32 UTF-8 bytes) for AES-256.
+# IVs must be 16 characters (16 UTF-8 bytes) for AES-CBC.
+# To generate compatible hex strings, use 16 raw bytes for a 32-character key:
+#   openssl rand -hex 16
+# and 8 raw bytes for a 16-character IV:
+#   openssl rand -hex 8
+# Production deployments must inject secure values through secret configuration.
+SHARE_TOKEN_AES_KEY=__INJECT_SECURE_32_CHARACTER_KEY__
+SHARE_TOKEN_AES_IV=__INJECT_SECURE_16_CHARACTER_IV__
+GIT_TOKEN_AES_KEY=__INJECT_SECURE_32_CHARACTER_KEY__
+GIT_TOKEN_AES_IV=__INJECT_SECURE_16_CHARACTER_IV__
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GIT_TOKEN_AES_IV key should go before the GIT_TOKEN_AES_KEY key

(UnorderedKey)